server/methods/ignoreUser.js

```diff
@@ -0,0 +1,29 @@
+/* globals RocketChat */
+
+Meteor.methods({
+	ignoreUser({rid, userId: ignoredUser, ignore = true}) {
+		check(ignoredUser, String);
+		check(rid, String);
```

Comment:
You're type checking everything but the `ignore`. Is there a good reason behind this or can it be type checked as well?

Reply:
just because the name suggest ignore and the params is a boolean so we can define a default... if you dont pass any information I will assume you really want ignore, if you pass`false` you want remove the ignore , I dont like the idea of have 2 methods to do the same thing...

Reply:
hmmm yep you are right :p